HowToAddNewLanguage.md

```diff
@@ -48,4 +47,10 @@ ignore = [
 ]
 ```
 
+## Add New Translation Tags
+
+- **Important**: If you add any new translation tags, they must first be added to the `messages_en_GB.properties` file. This ensures consistency across all language files.
+- New translation tags **must be added** to the `messages_en_GB.properties` file to maintain a reference for other languages.
+- After adding the new tags to `messages_en_GB.properties`, add and translate them in the respective language file (e.g., `messages_pl_PL.properties`).
```

Comment:
``After adding the new tags to `messages_en_GB.properties`, add and translate them in the respective language file (e.g., `messages_pl_PL.properties`).``

We are asking users to convert to 36 languages for any new additions?

Reply:
We don't actually need to worry about the introduction of the new script.